src/Configuration/ResourceManager.php

```diff
@@ -148,6 +152,27 @@ public function getPath($name)
 
         return $this->paths[$name];
     }
+    
+    /**
+     * Takes a known path with relative additional atoms and returns a new path
+     * for instance constructRelativePath('public/images/example')
+     *
+     * @return AbsoluteUnixPath Object
+     **/
+    public function constructRelativePath($name)
+    {
+        list($firstAtom) = explode('/', $name);
```

Comment:
This is only an access interface, if you ask for getPath("cache/test") you will still get back an OS specific path, just we're using forward slashes as a divider, the same as we do in config->get('general/branding')